### PR TITLE
Spelling batch 8

### PR DESCRIPTION
**Pull request by kingthorin**
Per: https://github.com/zaproxy/zap-extensions/pull/3083#issuecomment-1368085064 and https://github.com/check-spelling/zap-extensions/actions/runs/3808821577#summary-10368675301.

I've still left occurrences of "GitHub" or "JavaScript" not following the caps expectations (I'm not sure they're worth changing). There's also still a ton of stuff in the bruteforce sittinglittleducks package(s).

* the the
* should should
* would would
* corresponding corresponding
* may may
* that that
* return return
* during during
* element element
* COMTEXT > CONTEXT
* THREHOLD > THRESHOLD
* locl > lock
* implemention > implementation
* paremtheses > parentheses
* Boolen > Boolean

- tweak one set of short wrapped lines

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java [hunk 1]
```diff
@@ -602,8 +602,7 @@ public void scan() {
                 // if we didn't hit something with one of the iterations for the parameter (ie, if
                 // the output when changing the param is stable),
                 // check if the parameter might be vulnerable by comparing its LCS with the
-                // original
-                // LCS for a valid login
+                // original LCS for a valid login
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java:605]: This is really unnecessary.

File: addOns/authstats/src/main/javahelp/org/zaproxy/zap/extension/authstats/resources/help/contents/authStats.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <H1>Authentication Statistics</H1>
 The icon for this add-on was derived from the
```
Comment (thc202) [inline addOns/authstats/src/main/javahelp/org/zaproxy/zap/extension/authstats/resources/help/contents/authStats.html:38]: The changelog should be updated.

File: addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java [hunk 3]
```diff
@@ -44,7 +44,7 @@ public class AddAscanRuleDialog extends StandardFieldsDialog {
```
Comment (thc202) [inline addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddAscanRuleDialog.java:44]: The changelog should be updated.

File: addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/CookieUtils.java [hunk 4]
```diff
@@ -50,7 +50,7 @@ private CookieUtils() {
      *
```
Comment (thc202) [inline addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/CookieUtils.java:50]: The changelog should be updated.

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/GitMetadata.java [hunk 5]
```diff
@@ -525,7 +525,7 @@ public byte[] getObjectData(
                     }
                 }
                 // final sanity check, if all of the above panned out for version 1 index file.
-                // Note: we *think* that that "pack index" file version 1 is compatible with "pack"
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/GitMetadata.java:528]: Wasn't the second `that` to be a `the`?

Comment (kingthorin) [reply addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/GitMetadata.java:528]: Looks like it. Will tweak

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java (hunk 1 above)
Comment (kingthorin) [reply addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java:605]: Yes it’s only a whitespace change but it does make it read easier

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

**Pull request merged**